lib/mlxdevm.c

```diff
@@ -270,6 +270,38 @@ int mlxdevm_port_fn_state_get(struct mlxdevm *dl, struct mlxdevm_port *port,
 	return 0;
 }
 
+static int cmd_netdev_get_cb(const struct nlmsghdr *nlh, void *data)
+{
+	struct genlmsghdr *genl = mnl_nlmsg_get_payload(nlh);
+	struct nlattr *tb[MLXDEVM_ATTR_MAX + 1] = {};
+	const char *ifname = NULL;
+
+	mnl_attr_parse(nlh, sizeof(*genl), attr_cb, tb);
+	if (!tb[MLXDEVM_ATTR_DEV_BUS_NAME] || !tb[MLXDEVM_ATTR_DEV_NAME] ||
+	    !tb[MLXDEVM_ATTR_PORT_INDEX])
+		return MNL_CB_ERROR;
+
+	ifname = mnl_attr_get_str(tb[MLXDEVM_ATTR_PORT_NETDEV_NAME]);
+	strcpy((char *)data, ifname);
+	return MNL_CB_OK;
+}
+
+int mlxdevm_port_fn_ifname_get(struct mlxdevm *dl, struct mlxdevm_port *port,
```

Comment:
@BodongWang , rename as port_netdev_get().

because this is attached to the port instead of function.